src/client/dfs/dfs_internal.c

```diff
@@ -259,6 +275,40 @@ dfs_hdl_insert(const char *str, int type, daos_handle_t *oh, struct dfs_mnt_hdls
 	return rc;
 }
 
+int
+dfs_hdl_cont_destroy(const char *pool, const char *cont, bool force)
+{
+	d_list_t		*rlink = NULL;
+	struct dfs_mnt_hdls	*hdl;
+	char			cont_pool[DAOS_PROP_LABEL_MAX_LEN * 2 + 1];
+	daos_size_t		len;
+
+	D_ASSERT(pool);
+	len = snprintf(cont_pool, strlen(pool) + strlen(cont) + 2, "%s/%s", pool, cont);
+	D_ASSERT(len >= strlen(pool) + strlen(cont) + 1);
+
+	if (coh_hash == NULL)
+		return 0;
```

Comment:
Minor suggestion: you could move these two lines above (before "D_ASSERT(pool)").

Reply:
sure will do that too in the follow on PR. thx!